eos-migrate-firefox-profile.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Migrate Firefox profile to new path
+
+ConditionPathIsDirectory=%h/.mozilla
+ConditionPathIsDirectory=%h/.var/app/org.mozilla.Firefox
+ConditionPathIsDirectory=!%h/.var/app/org.mozilla.Firefox/.mozilla
+
+[Service]
+Type=simple
+ExecStart=/usr/lib/eos-boot-helper/eos-migrate-firefox-profile
+Restart=no
```

Comment:
I think for a script where you want the unit to not be considered started until the script completes, you would want `Type=oneshot`. To persist it, add `RemainAfterExit=yes`. See [systemd.service(5)](https://man7.org/linux/man-pages/man5/systemd.service.5.html). With `Type=Simple`, systemd will consider the unit started as soon as it finishes forking the script. That means `gnome-session.target` could be considered started before this script completes.